controller/controller_server/csi_controller_server.py

```diff
@@ -75,17 +76,41 @@ def GetCapacity(self, request, context):
     def ControllerGetCapabilities(self, request, context):
         logger.debug("ControllerGetCapabilities")
         types = csi_pb2.ControllerServiceCapability.RPC.Type
-                    
+        
+        context.set_code(grpc.StatusCode.OK)
         return csi_pb2.ControllerGetCapabilitiesResponse(
            capabilities=[csi_pb2.ControllerServiceCapability(
                             rpc=csi_pb2.ControllerServiceCapability.RPC(type=types.Value("CREATE_DELETE_VOLUME"))),
                          csi_pb2.ControllerServiceCapability(
                              rpc=csi_pb2.ControllerServiceCapability.RPC(type=types.Value("PUBLISH_UNPUBLISH_VOLUME")))  ])
  
+    def GetPluginInfo(self, request, context):
+        logger.debug("GetPluginInfo")
+        context.set_code(grpc.StatusCode.OK)
```

Comment:
Not sure you need to do that, because by default its OK.
I think only for error you should set the grpc status code.  (relevant for other places in the code here). 

if in the code we should raise error(e.g  For example if no name in conf file) then u can use context.abort(grpc.StatusCode.<FAIL_CONST>) which raises the relevant grpc error.  and then also use it in the UT to check if exception raised.